src/chat.rs

```diff
@@ -1778,6 +1794,56 @@ pub fn set_selfavatar_timestamp(
     Ok(())
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub enum MuteDuration {
+    NotMuted,
+    Forever,
+    MutedUntilTimestamp(i64),
+}
+
+impl MuteDuration {
+    // TODO use serde compatible functions?
+    pub fn serialize(&self) -> i64 {
+        match &self {
+            MuteDuration::NotMuted => 0,
+            MuteDuration::Forever => 1,
+            MuteDuration::MutedUntilTimestamp(timestamp) => *timestamp as i64, // TODO make this pretier?
```

Comment:
Maybe check if `timestamp > 1` here and return 0 otherwise.

Reply:
currently I exploit this function for testing (setting a timestamp in the past)